src/main/java/com/voicepin/flow/client/FlowClient.java

```diff
@@ -1,45 +1,176 @@
 package com.voicepin.flow.client;
 
-
-import com.voicepin.flow.client.calls.*;
+import com.voicepin.flow.client.calls.AddVoiceprintCall;
+import com.voicepin.flow.client.calls.Call;
+import com.voicepin.flow.client.calls.EnrollCall;
+import com.voicepin.flow.client.calls.GetVoiceprintCall;
+import com.voicepin.flow.client.calls.VerifyInitCall;
 import com.voicepin.flow.client.exception.FlowClientException;
-import com.voicepin.flow.client.request.*;
-import com.voicepin.flow.client.result.*;
+import com.voicepin.flow.client.request.AddVoiceprintRequest;
+import com.voicepin.flow.client.request.EnrollRequest;
+import com.voicepin.flow.client.request.GetVoiceprintRequest;
+import com.voicepin.flow.client.request.VerifyInitRequest;
+import com.voicepin.flow.client.request.VerifyRequest;
+import com.voicepin.flow.client.result.AddVoiceprintResult;
+import com.voicepin.flow.client.result.EnrollResult;
+import com.voicepin.flow.client.result.GetVoiceprintResult;
+import com.voicepin.flow.client.result.VerifyInitResult;
+import com.voicepin.flow.client.ssl.PinnedCertificateStrategy;
+import com.voicepin.flow.client.ssl.TrustedCertificateStrategy;
+import com.voicepin.flow.client.ssl.CertificateStrategy;
+import com.voicepin.flow.client.ssl.AnyCertificateStrategy;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
- * @author kodrzywolek
+ * @author kodrzywolek, Lukasz Warzecha
```

Comment:
Tu też by pasował jakiś krótki JavaDoc. Np:

Client for VoicePIN Flow REST API (Text Independent Biometric Voice Verification server).